src/sentry/api/serializers/models/user.py

```diff
@@ -1,6 +1,9 @@
 from collections import defaultdict
+from typing import Any, Callable, Dict, List, MutableMapping, Optional, Sequence, Union, cast
 
 from django.conf import settings
+from django.db.models import QuerySet
+from typing_extensions import TypedDict
```

Comment:
note that until we're on 3.8, we need to use this TypedDict instead of the default, as you cannot get optional fields at runtime with the 3.7 implementation